appnav/src/main/kotlin/io/element/android/appnav/room/LoadingRoomNodeView.kt

```diff
@@ -72,7 +72,8 @@ fun LoadingRoomNodeView(
                 modifier = Modifier
                     .fillMaxSize()
                     .padding(padding)
-                    .padding(16.dp), contentAlignment = Alignment.Center
+                    .padding(16.dp),
```

Comment:
I think it's better to call `.consumeWindowInsets(padding)` here than pass the child a modifier with window insets which it then has to ignore?

Reply:
Is this not done automatically by the scaffold? I'm just adding some paddings for the text to not be touching borders

Reply:
No, it changed a little while ago so Scaffold doesn't do it any more.

The [documentation for the `content` param](https://developer.android.com/reference/kotlin/androidx/compose/material3/package-summary#Scaffold(androidx.compose.ui.Modifier,kotlin.Function0,kotlin.Function0,kotlin.Function0,kotlin.Function0,androidx.compose.material3.FabPosition,androidx.compose.ui.graphics.Color,androidx.compose.ui.graphics.Color,androidx.compose.foundation.layout.WindowInsets,kotlin.Function1)) now says:

> The lambda receives a [PaddingValues](https://developer.android.com/reference/kotlin/androidx/compose/foundation/layout/PaddingValues) that should be applied to the content root via Modifier.padding and [Modifier.consumeWindowInsets](https://developer.android.com/reference/kotlin/androidx/compose/foundation/layout/package-summary#(androidx.compose.ui.Modifier).consumeWindowInsets(androidx.compose.foundation.layout.WindowInsets)) to properly offset top and bottom bars.

Reply:
Yeah, https://github.com/vector-im/element-x-android/commit/848f1cb98868f60f4c83888809c76f4749209a22 changed all the existing uses at that point (along with a bunch of other unrelated changes :sweat:)